relayer-cli/src/commands/tx/upgrade.rs

```diff
@@ -323,11 +324,11 @@ mod tests {
     fn test_upgrade_chain_no_height_offset() {
         assert!(TxIbcUpgradeChainCmd::try_parse_from(&[
             "test",
-            "--receiver-chain",
+            "--reference-chain",
             "chain_receiver",
-            "--sender-chain",
+            "--sendhostin",
```

Comment:
This should probably be:

```suggestion
            "--host-chain",
```

or is there a reason for this flag name here and in the other tests?

Reply:
ea56ae0fc84a16a2bf91c5f766141ffcffaca16b